prometheus/src/main/scala/fr/davit/akka/http/metrics/prometheus/PrometheusSettings.scala

```diff
@@ -73,21 +74,22 @@ final case class PrometheusSettings(
     includeMethodDimension: Boolean,
     includePathDimension: Boolean,
     includeStatusDimension: Boolean,
+    includeCustomLabels: Seq[CustomDimension] = Seq.empty,
```

Comment:
`Seq` is immutable only from scala `2.13`. prefer using `immutable.Seq` to endure immutability for previous scala versions